app/common/view.c

```diff
@@ -68,6 +69,26 @@ void h_initialize(__Z_UNUSED unsigned int _) {
     UX_WAIT();
 }
 
+uint8_t getIntroPages() {
+#if defined(APP_BLIND_MODE_ENABLED)
+    if (mustReply == REVIEW_ADDRESS || !app_mode_blind()) {
+        #if defined(TARGET_NANOS)
+            return INTRO_PAGES - 1;
+        #else
+            return INTRO_PAGES;
+        #endif
+    } else {
+        return INTRO_PAGES;
+    }
+#else
+    return INTRO_PAGES;
+#endif
+}
+
+bool h_paging_intro_screen() {
+    return viewdata.itemIdx < getIntroPages();
```

Comment:
We've tested in other apps without including these new features and it's working fine! itemIdx = 0 will return the first page of the transaction (if review pages and blind signing is disabled). Otherwise, it returns the intro page that correspond